Frontend/ui-driver/src/Helpers/Utils.js

```diff
@@ -853,6 +853,54 @@ export const initiatePP = function () {
     console.log("%cHyperpay initiate Result - Already Initiated", "background:darkblue;color:white;font-size:13px;padding:2px", window.__payload);
   }
 }
+export const getAvailableUpiApps = function (resultCb) {
+  var cb = function (code) {
+    return function (_response) {
+      return function () {
+        console.log("%cUPIINTENT Terminate Response ", "background:darkblue;color:white;font-size:13px;padding:2px", code, _response);
+      }
+    }
+  }
+  if (JOS) {
+    try {
+      var result = function (code) {
+        return function (_response) {
+          return function () {
+            console.log("%cUPIINTENT initiate Result ", "background:darkblue;color:white;font-size:13px;padding:2px", _response);
+            try {
+              let resultPayload = JSON.parse(_response)
+              resultCb(resultPayload.payload.response.available_apps)();
+              killPP();
+            } catch (err) {
+              console.log("%cUPIINTENT initiate Result error", "background:darkblue;color:white;font-size:13px;padding:2px", err, code);
+            }
+          }
+        }
+      }
+      var payload = {
+        "UPI_PAYMENT_METHOD": "NA",
+        "client_id": "nammayatriprovider",
```

Comment:
can we make this configurable? How will handle it in case of other merchants?

Reply:
we can make this configurable. it is not required to be configurable because even if we send `""` then also it will work. It is add because it is mandatory field.